Common/Python/PandasData.cs

```diff
@@ -134,8 +135,8 @@ public PandasData(object data)
 
             var type = data.GetType();
             _isFundamentalType = type == typeof(Fundamental);
-            IsCustomData = type.Namespace != typeof(Bar).Namespace;
             _symbol = ((IBaseData)data).Symbol;
+            IsCustomData = type.Namespace != typeof(Bar).Namespace || Extensions.GetCustomDataTypeFromSymbols(new Symbol[] { _symbol }) != null ? true : false;
```

Comment:
1. Only types in `QuantConnect.Data.Market` are cosidered "not custom"? What about `QuantConnect.Data`?
2. nit - the ternary operation might not be necessary: `!= null` is enough.
3. This check is also in `Algorithm/QCAlgorithm.History.cs:1087`: maybe move it to a single location like Extensions for single source of truth

Reply:
1. The other data types present in QuantConnect.Data are `BaseData` and `DynamicData`, but they are abstract, so we cannot instantiate an object of those classes. Then, it seems that the only custom types are the ones not present in `QuantConnect.Data.Market`.